.eslintignore

```diff
@@ -1,5 +1,12 @@
-.circleci/*
-.githooks/*
-.idea/*
-.vscode/*
-node_modules/*
+.circleci
+.githooks
+.idea
+.vscode
+.yarn*
+dist
+node_modules
+*.d.ts
+*.js
```

Comment:
While this likely to be true going forward, I think it would be safer to just list the files to ignore, since there are a small number.

Reply:
The linter gave some warnings about parsing .js files when it wasn't expecting them but, as above, adding `--ext .ts` seems to operate better than using an ignore statement in this case.